CHANGELOG.md

```diff
@@ -99,6 +99,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Fixed
 - Fix flaky tests in CloseIndexIT by addressing cluster state synchronization issues ([#18878](https://github.com/opensearch-project/OpenSearch/issues/18878))
+- Fix flaky ExistsQueryBuilderTests.testToQuery ([#18274](https://github.com/opensearch-project/OpenSearch/issues/18274))
```

Comment:
We generally don't add changelog entries for non-user facing things like test fixes. There is some guidance [here](https://github.com/opensearch-project/OpenSearch/blob/d9a927464ec62634d520bc32bbd34104c4f4c097/CONTRIBUTING.md#changelog).

Reply:
Fixed